pyproject.toml

```diff
@@ -0,0 +1,134 @@
+[project]
+
+authors = [{name = "Jeffrey Larson"}, {name = "Stephen Hudson"},
+            {name = "Stefan M. Wild"}, {name = "David Bindel"},
+            {name = "John-Luke Navarro"}]
+
+dependencies = ["pip>=24.3.1,<25", "setuptools>=75.1.0,<76", "numpy>=1.21,<3",
```

Comment:
"pip>=24.3.1,<25", "setuptools>=75.1.0,<76",
should prob be below in build systems so separate build from run deps.